gctscript/wrappers/gct/gctwrapper_test.go

```diff
@@ -186,18 +187,12 @@ func TestExchangePairs(t *testing.T) {
 func TestAccountInfo(t *testing.T) {
 	t.Parallel()
 	_, err := gct.ExchangeAccountInfo()
-	if !errors.Is(err, objects.ErrWrongNumArguments) {
-		t.Fatal(err)
-	}
+	require.ErrorIs(t, err, objects.ErrWrongNumArguments)
 	obj, err := gct.ExchangeAccountInfo(ctx, exch, assetType)
-	if err != nil {
-		t.Fatalf("received: %v but expected: %v", err, nil)
-	}
-	rString, _ := objects.ToString(obj)
-	if rString != `error: "Bitstamp REST or Websocket authentication support is not enabled"` {
-		t.Errorf("received: %v but expected: %v",
-			rString, `error: "Bitstamp REST or Websocket authentication support is not enabled"`)
-	}
+	require.NoError(t, err)
+	rString, ok := objects.ToString(obj)
+	require.True(t, ok)
```

Comment:
🐒 Errors which aren't self evident should have comments.
Normal Criteria for not needing a comment:
* Testing the function named in the unit tests
* `NoError`, `NotEmpty` or `ErrorIs` against the return value of that function
* Most other things would benefit from giving context for future maintainers 🙏 
```suggestion
	require.True(t, ok, "ExchangeAccount return value must return correctly from objects.ToString")
```

Reply:
"ExchangeAccount return value must return correctly from objects.ToString"
or "ExchangeAccount**Info** return value must return correctly from objects.ToString"

Reply:
Sorry for the delay replying.
You're right, the later: `"ExchangeAccountInfo return value must return correctly from objects.ToString"`
Also spotting that the func itself should be renamed to match: `TestExchangeAccountInfo`